### PR TITLE
Add file mode for crontab and cron.<file>

### PR DESCRIPTION
**Pull request by maadhavgannamanthir**
Added file mode permissions for 
/etc/crontab
/etc/cron.<files>

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# @summary This class ensures that the distro-appropriate cron package is installed. This class should not be used directly under normal circumstances. Instead, use the *cron* class.
+# @summary Installs the cron package and ensures it is up to date. # lint:ignore:140chars
```
Comment (anders-larsson) [inline manifests/install.pp:1]: Where this change come from? It seems unrelated to your PR.

File: manifests/init.pp [hunk 2]
```diff
@@ -45,6 +47,8 @@
   String[1]            $crontab_mailto          = 'root',
   Optional[Stdlib::Absolutepath] $crontab_home  = undef,
   Cron::Run_parts      $crontab_run_parts       = {},
+  Stdlib::Filemode     $crontab_file_mode       = '0600',
+  Stdlib::Filemode     $crontab_run_parts_mode  = '0644',
```
Comment (anders-larsson) [inline manifests/init.pp:51]: Use 0644 / 0755 (current defaults) for the parameters instead, to keep the current behaviour.

**Comment by maadhavgannamanthir** [comment]
> Spec tests has not been updated. It would probably fail dye to the changed permissions of these files.
> 
> Add tests for these parameters.

Updated parameters in Spec

File: manifests/init.pp (hunk 2 above)
Comment (maadhavgannamanthir) [reply manifests/init.pp]: Corrected mode permissions

File: manifests/install.pp (hunk 1 above)
Comment (maadhavgannamanthir) [reply manifests/install.pp]: Checkout to previous file.

**Comment by Henrik-Hansson** [comment]
Can you combine all 10 commits to one commit?
